.github/workflows/image_build_push.yml

```diff
@@ -0,0 +1,48 @@
+# This is a basic workflow to help you get started with Actions
+
+name: docker image
+
+# Controls when the action will run. Triggers the workflow on push or pull request 
+# events but only for the master branch
+on:
+  push:
+    branches: [ master, gis-based-mode-detection, join_redirect_to_static, image_CI ]
```

Comment:
`image_CI` was the name of the branch I was developing on in my fork. It was included so that the workflow would run on that branch and I wouldn't have to make or merge changes to one of the branches we care about. I can get rid of it without introducing any problem to how the workflow works. That said, I would probably create a new branch again if we wanted to make changes to this workflow. Maybe it is nice to have around? Let me know if this if a bad idea.

Reply:
For the record, it is better to submit PRs from branches and not from master.
And yes, we shouldn't have references to ghost branches committed to master.

Reply:
Makes sense. I think I normally try to do that. Here I merged into master to see the workflow working on a merge and just created the PR from there when it finished.